test/stampit-specs.js

```diff
@@ -87,28 +70,36 @@ test('stampit().methods(a, b)', function () {
     'Should mixIn objects when multiple methods are passed.');
 });
 
+module('Basics State');
+
 test('stampit({}, state)', function () {
-  var obj = stampit({}, {
-    foo: {bar: 'bar'}
-  }).create();
+  var obj = stampit({}, { foo: { bar: 'bar' } }).create();
+  
   equal(obj.foo.bar, 'bar',
     'Should set default state.');
 });
 
 test('stampit().state()', function () {
-  var obj = stampit().state({
-    foo: {bar: 'bar'},
-    stateOverride: false
+  var obj = stampit(null, {
```

Comment:
This is also now testing both the parameter form and the `.state()` form. Why are you introducing these test ambiguities?